src/sage/schemes/elliptic_curves/ell_field.py

```diff
@@ -207,21 +221,31 @@ def quadratic_twist(self, D=None):
                 raise ValueError("twisting parameter D must be nonzero when characteristic is not 2")
 
         if char != 2:
-            b2,b4,b6,b8 = self.b_invariants()
+            b2,b4,b6,_ = self.b_invariants()
             # E is isomorphic to  [0,b2,0,8*b4,16*b6]
-            return EllipticCurve(K,[0,b2*D,0,8*b4*D**2,16*b6*D**3])
-
-        # now char==2
-        if self.j_invariant() != 0: # iff a1!=0
-            a1,a2,a3,a4,a6 = self.ainvs()
-            E0 = self.change_weierstrass_model(a1,a3/a1,0,(a1**2*a4+a3**2)/a1**3)
-            # which has the form = [1,A2,0,0,A6]
-            assert E0.a1() == K(1)
-            assert E0.a3() == K(0)
-            assert E0.a4() == K(0)
-            return EllipticCurve(K,[1,E0.a2()+D,0,0,E0.a6()])
+            Et = EllipticCurve(K,[0,b2*D,0,8*b4*D**2,16*b6*D**3])
```

Comment:
Some whitespace around `,` and `8 * b4 `... is normally what linting likes, right?